policy-controller/grpc/src/outbound.rs

```diff
@@ -324,31 +316,39 @@ fn convert_outbound_http_route(
 
     let rules = rules
         .into_iter()
-        .map(|HttpRouteRule { matches, backends }| {
-            let backends = backends
-                .into_iter()
-                .map(convert_http_backend)
-                .collect::<Vec<_>>();
-            let dist = if backends.is_empty() {
-                outbound::http_route::distribution::Kind::FirstAvailable(
-                    outbound::http_route::distribution::FirstAvailable {
-                        backends: vec![outbound::http_route::RouteBackend {
-                            backend: Some(backend.clone()),
-                            filters: vec![],
-                        }],
-                    },
-                )
-            } else {
-                outbound::http_route::distribution::Kind::RandomAvailable(
-                    outbound::http_route::distribution::RandomAvailable { backends },
-                )
-            };
-            outbound::http_route::Rule {
-                matches: matches.into_iter().map(http_route::convert_match).collect(),
-                backends: Some(outbound::http_route::Distribution { kind: Some(dist) }),
-                filters: Default::default(),
-            }
-        })
+        .map(
+            |HttpRouteRule {
+                 matches,
+                 backends,
+                 request_timeout,
+             }| {
+                let backends = backends
+                    .into_iter()
+                    .map(convert_http_backend)
+                    .collect::<Vec<_>>();
+                let dist = if backends.is_empty() {
+                    outbound::http_route::distribution::Kind::FirstAvailable(
+                        outbound::http_route::distribution::FirstAvailable {
+                            backends: vec![outbound::http_route::RouteBackend {
+                                backend: Some(backend.clone()),
+                                filters: vec![],
+                                request_timeout: None,
```

Comment:
Do we also need to store the backend_request_timeout on the route rule so that we can apply it to the default backend here?

Reply:
oh, good point, will do!